config/source.json

```diff
@@ -303,6 +303,16 @@
             "path": "LICENSE"
         }
     },
+    "libaom": {
+        "type": "git",
+        "rev": "main",
```

Comment:
maybe should we use the latest tag instead?

Reply:
Indeed. But I haven't figured out how to get the latest tag yet. Maybe we need to customize the XML parsing of `https://storage.googleapis.com/aom-releases/` .